plugins/modules/infra_host.py

```diff
@@ -0,0 +1,429 @@
+            type: str
+            returned: Always
+        host_version:
+            description:
+                - "The version of the Host platform services."
+            type: str
+            returned: Always
+        id:
+            description:
+                - "The resource identifier."
+            type: str
+            returned: Always
+        ip_address:
+            description:
+                - "The IP address of the Host."
+            type: str
+            returned: Always
+        ip_space:
+            description:
+                - "The IP Space of the Host."
+            type: str
+            returned: Always
+        legacy_id:
+            description:
+                - "The legacy Host object identifier."
+            type: str
+            returned: Always
+        location_id:
+            description:
+                - "The resource identifier."
+            type: str
+            returned: Always
+        mac_address:
+            description:
+                - "The MAC address of the Host."
+            type: str
+            returned: Always
+        maintenance_mode:
+            description: ""
+            type: str
+            returned: Always
+        nat_ip:
+            description:
+                - "The NAT IP address of the Host."
+            type: str
+            returned: Always
+        noa_cluster:
+            description:
+                - "The CSP cluster identifier (internal use only)."
+            type: str
+            returned: Always
+        ophid:
+            description:
+                - "The unique On-Prem Host ID generated by the On-Prem device and assigned to the Host once it is registered and logged into the Infoblox Cloud."
+            type: str
+            returned: Always
+        pool_id:
+            description:
+                - "The resource identifier."
+            type: str
+            returned: Always
+        serial_number:
+            description:
+                - "The unique serial number of the Host."
+            type: str
+            returned: Always
+        tags:
+            description:
+                - "Tags associated with this Host."
+            type: dict
+            returned: Always
+        timezone:
+            description:
+                - "The timezone of the Host."
+            type: str
+            returned: Always
+        updated_at:
+            description:
+                - "The timestamp of the latest update on Host."
+            type: str
+            returned: Always
+"""  # noqa: E501
+
+from ansible_collections.infoblox.bloxone.plugins.module_utils.modules import BloxoneAnsibleModule
+
+try:
+    from bloxone_client import ApiException, NotFoundException
+    from infra_mgmt import Host, HostsApi
+except ImportError:
+    pass  # Handled by BloxoneAnsibleModule
+
+
+class HostsModule(BloxoneAnsibleModule):
```

Comment:
nit: should be HostModule? Also, i am thinking if it should be `InfraHostModule`, since we also have dns host, dhcp host, etc.  
```suggestion
class HostModule(BloxoneAnsibleModule):
```

Reply:
Fixed